frontend/src/model/settings/ModelAccess.tsx

```diff
@@ -143,7 +145,13 @@ export default function ModelAccess({ model }: ModelAccessProps) {
         </TableHead>
         <TableBody>{accessListEntities}</TableBody>
       </Table>
-      <LoadingButton variant='contained' aria-label='Save access list' onClick={updateAccessList} loading={loading}>
+      <LoadingButton
```

Comment:
If this is disabled due to the `roleError` value being set to `true` I would add some error text below the button explaining the error. 

It might be more work, but you could replace the boolean and have it as a string, and when you set `roleError` inside `UserDisplay` set it to error message created by `isUserInformationError`.